tools/dds/dds-server/lrs-device-watcher.h

```diff
@@ -14,8 +14,8 @@ class lrs_device_watcher : public std::enable_shared_from_this<lrs_device_watche
 public:
     lrs_device_watcher( rs2::context &_ctx );
     ~lrs_device_watcher();
-    void run( std::function< void( rs2::device ) > add_device_cb,
-              std::function< void( rs2::device ) > remove_device_cb );
+    void run( std::function< void( rs2::device && ) > add_device_cb,
```

Comment:
Any special reason? `rs2::device` has a shared_ptr inside so it just does a reference count

Reply:
Did not realize it was just hiding a shared_ptr and thought it would be more efficient. Will roll back the changes

Reply:
Done